frontend/src/metabase/databases/components/DatabaseEngineField/DatabaseEngineWidget.tsx

```diff
@@ -138,17 +160,38 @@ const EngineSearch = ({
 
   return (
     <EngineSearchRoot role="combobox">
-      <Input
-        value={searchText}
-        placeholder={t`Search for a database…`}
-        autoFocus
-        aria-autocomplete="list"
-        aria-controls={getListBoxId(rootId)}
-        aria-activedescendant={getListOptionId(rootId, activeOption)}
-        fullWidth
-        onChange={handleSearch}
-        onKeyDown={handleKeyDown}
-      />
+      <FieldRoot orientation="vertical" alignment="end">
+        <FieldLabelContainer orientation="vertical" hasDescription={false}>
+          <FieldLabel htmlFor="search-for-a-database" hasError={false}>
+            {t`Search for a database`}
+          </FieldLabel>
+        </FieldLabelContainer>
+        <Input
+          id="search-for-a-database"
+          value={searchText}
+          autoFocus
+          aria-autocomplete="list"
+          aria-controls={getListBoxId(rootId)}
+          aria-activedescendant={getListOptionId(rootId, activeOption)}
+          fullWidth
+          onChange={handleSearch}
+          onKeyDown={handleKeyDown}
+        />
+        <div
+          className="visually-hidden"
+          aria-live="polite"
+          aria-atomic="true"
+          style={{
+            position: "absolute",
+            left: "-9999px",
+            width: "1px",
+            height: "1px",
+            overflow: "hidden",
+          }}
+        >
+          {dbOptionsAriaMessage}
+        </div>
```

Comment:
instead of an inline div with a bunch of inline style props, could this be reusable component that could be helpful throughout the app, like `<HiddenAriaMessage message={dbOptionsMessage} />` ?